R-packages/evalcast/R/get_predictions.R

```diff
@@ -44,19 +45,22 @@ get_predictions <- function(forecaster,
                             incidence_period,
                             ahead,
                             geo_type,
-                            geo_values = "*") {
+                            geo_values = "*",
+                            ...) {
   assert_that(is_tibble(signals), msg="`signals` should be a tibble.")
+  params <- list(...)
   forecast_dates %>%
-    map(~ get_predictions_single_date(
-      forecaster = forecaster,
-      name_of_forecaster = name_of_forecaster,
-      signals = signals,
-      forecast_date = .x,
-      incidence_period = incidence_period,
-      ahead = ahead,
-      geo_type = geo_type,
-      geo_values = geo_values
-    )) %>%
+    map(~ do.call(
+          get_predictions_single_date,
+          c(list(forecaster = forecaster,
+                 name_of_forecaster = name_of_forecaster,
+                 signals = signals,
+                 forecast_date = .x,
+                 incidence_period = incidence_period,
+                 ahead = ahead,
+                 geo_type = geo_type,
+                 geo_values = geo_values),
+            params))) %>%
```

Comment:
Just curious: Why do you use `do.call` here rather than the simpler choice of adding `...` as the last argument to `get_predictions_single_date`?

Reply:
There's some very weird issues that happen with `map()` and dots (I couldn't get what you're suggesting to work, and many variants of your suggestion as well).

Reply:
I see, thanks!